slurm_gpu/gpu_binding.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""
+script to calculate GPU binding
+
+author: Samuel Moors (Vrije Universiteit Brussel)
+"""
+import math
+import os
+import pynvml
+import json
+
+
+def gpu_cpu_affinity(gpuid):
+    """
+    get GPU-CPU affinity
+    returns a list of allocated CPU ids that are on the same socket as gpuid
+    """
+    cpusetsize = math.ceil(os.cpu_count() / 64)
+    handle = pynvml.nvmlDeviceGetHandleByIndex(gpuid)
+    aff_str = ''
+    for c_aff in pynvml.nvmlDeviceGetCpuAffinity(handle, cpusetsize):
+        aff_string = f'{c_aff:064b}{aff_str}'
```

Comment:
you're overwrite the same var? This does need to be `aff_str = `?

Reply:
indeed, fixed